documenting.rst

```diff
@@ -1565,11 +1565,12 @@ You can also use ``make help`` to see a list of targets supported by
 
 See also :file:`Doc/README.rst` for more information.
 
+Using sphinx-build
+------------------
 
-Without make
-------------
-
-Install the Sphinx, blurb, and python-docs-theme packages from PyPI.
+Sometimes we directly want to execute the sphinx-build tool, in this case, you can
```

Comment:
"tool instead of through ``make`` (although the latter is still the preferred way). In this case,"""